static/app/utils/useApiRequests.tsx

```diff
@@ -172,114 +151,39 @@ function useApiRequests({
 
   const [state, setState] = useState<State>({...initialState});
```

Comment:
```suggestion
  const [state, setState] = useState<State>(initialState);
```

Reply:
This is probably safe. @NisanthanNanthakumar do you know why you had this spread in here?

Reply:
I understand, but in this case, it would be fine